Test Automation Course/src/topic8/ServiciuStoc.java

```diff
@@ -0,0 +1,24 @@
+package topic8;
+
+import java.util.ArrayList;
+
+public class ServiciuStoc {
+    public static void main(String[] args) {
+        Carte Nr1 = new Carte("Titlu1", "Autor1", 1, 100);
```

Comment:
Deja sugerez sa practici unn pic colectiile si sa declari cartile deja intro lista de ex:
List<Carte> = new ArrayList<>();
si cu ajutorul la un loop sa sa adaugi in lista toate aceste carti ;)